config/dev/elastic-psp.yaml

```diff
@@ -46,3 +46,54 @@
         - min: 1
           max: 65535
     readOnlyRootFilesystem: false
+---
```

Comment:
Maybe add a comment to explain why we need a dedicated psp and what privileges are needed ?